README.md

```diff
@@ -1,5 +1,150 @@
 # ManimPango
 
-![Build Wheels](https://github.com/ManimCommunity/manimpango/workflows/Build%20Wheels/badge.svg)
+<p align="center">
+    <a href="https://pypi.org/project/manimpango/"><img src="https://img.shields.io/pypi/v/manimpango.svg?style=flat&logo=pypi" alt="PyPI Latest Release"></a>
+    <a href="https://pypi.org/project/manimpango/"><img alt="PyPI - Wheel" src="https://img.shields.io/pypi/wheel/manimpango"></a>
+    <a href="https://pypi.org/project/manimpango/"><img alt="PyPI - Downloads" src="https://img.shields.io/pypi/dm/manimpango"></a>
+    <a href="http://choosealicense.com/licenses/mit/"><img alt="PyPI - License" src="https://img.shields.io/pypi/l/manimpango"></a>
+    <a href="https://pypi.org/project/manimpango/"><img alt="PyPI - Python Version" src="https://img.shields.io/pypi/pyversions/pangocffi.svg?style=flat"></a>
+    <br>
+    <img src="https://github.com/ManimCommunity/manimpango/workflows/Build%20Wheels/badge.svg">
+</p>
 
-Small binding using Cython for Pango. Used internally with Manim.
+**ManimPango** is a C binding for [Pango](https://pango.gnome.org/) using [Cython](https://cython.org/), which is internally used in [Manim](https://www.manim.community) to render **Text**.
+
+
+
+## INSTALL
+
+Installing **ManimPango** is super easy, just use pip to install it. It is [`manimpango`](https://pypi.org/project/manimpango/) in PyPi.
+
+```sh
+pip install manimpango
+```
+
+#### Note:
+
+For **Linux Users**, installing the binary wheel provided in PyPi could cause some weird things happen to your system.
+
+For example, you would get a folder named `fontconfig`, in your directory, and it will have a lot of cache files each time you run Manim. To avoid those kind of weird things from happening, you would have to install it from source using the below command.
```

Comment:
```suggestion
For example, a folder named `fontconfig` might be created in the current working directory. That folder would then have a lot of cache files each time you run Manim. In order to avoid this, you will have to install **Manimpango** from source:.
```